mimic/canned_responses/maas_test_check.py

```diff
@@ -0,0 +1,67 @@
+"""
+Canned response for /test-check
+"""
+
+import collections
+
+
+def test_check(checkType):
+    """
+    Support test-check API
+    """
+    test_check_responses = collections.defaultdict(lambda: "")
+    test_check_responses["agent.cpu"] = [
+        {
+            "timestamp": 0,
+            "available": True,
+            "status": "success",
+            "metrics": {
+                "user_percent_average": {
+                    "data": 7.1695253641707,
```

Comment:
Also, important to note, `core.rng` should be a `random.Random(...)` instance owned by `MimicCore`, and not just a reference to the `random` module, since the `random` module's seed is global mutable shared state and therefore unpredictable in the presence of any other libraries which might use it to generate a random number.